account_consolidation/tests/test_account_consolidation.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields
+from odoo.tests.common import TransactionCase
+
+
+class TestAccountConsolidation(TransactionCase):
+
+    def setUp(self):
```

Comment:
I see a lot of write/create: if you don't need this to run for every tests: move the test case to `SavepointCase` and the setup to `setUpClass` to improve tests performances.

Reply:
again :)

Reply:
For this case, we need the `setUp()` to be called before each consolidation test to ensure existing `account.move` aren't impacted by other tests.